### PR TITLE
merge database branch with main

### PR DESCRIPTION
**Pull request by EduardoLopezPalomo**

### CODE REVIEW

File: server/bin/defaultdata.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        "postalcode" : "example2",
+        "city":"Seul",
+        "country": "korea",
+        "email": "example2@gmail.com",
+        "password": "password",
+        "admin": false,
+        "profileImage": []
+    }
+];
+let assocaitionAux = {
```
Comment (jannejjj) [inline server/bin/defaultdata.js:39]: Typo (should be association)

**Review by jannejjj — COMMENTED** [review]
Looks okay in general, just not sure 100% about what the member_event and member_association models are for

**Review by jannejjj — APPROVED** [review]

**Review by jannejjj — APPROVED** [review]

**Comment by jannejjj** [comment]
[Implement the database](https://trello.com/c/LMUJ3BMg/21-implement-the-database)

**Review by Enlarem — COMMENTED** [review]
I think everything is ok , but how can we test the "insertSampleData" function ?

**Comment by EduardoLopezPalomo** [comment]
the funtion is imported in the app.js file, so when you start the app it will load the examples automaticly

**Review by Enlarem — APPROVED** [review]

**Pull request merged**